etc/gwenview.profile

```diff
@@ -27,7 +28,7 @@ seccomp
 shell none
 tracelog
 
-private-bin gwenview,kbuildsycoca4,gimp,gimp-2.8
+private-bin gwenview,kbuildsycoca4,kbuildsycoca5,gimp,gimp-2.8
```

Comment:
I'm sorry, it's me again :)
For me Gwenview works fine as it is now, that is without kbuildsycoca5.... though I actually don't understand why it works.

The sycoca5 database seems to live in /home/user/.cache, where it is accessible from inside the sandbox. If the database is hidden (whitelisted profile, mkdir ~/.cache) it is recreated, but surprisingly this happens whether kbuildsycoca5 is available in the sandbox or not. This is on Debian.